packages/vscode-extension/lib/network/AsyncBoundedResponseBuffer.test.ts

```diff
@@ -79,15 +79,15 @@ describe("AsyncBoundedResponseBuffer", () => {
     const result1 = await buffer.put(requestId1, Promise.resolve(responseData1));
     const result2 = await buffer.put(requestId2, Promise.resolve(responseData2));
     assert.strictEqual(result1, true);
-    assert.strictEqual(result2, false);
+    assert.strictEqual(result2, true);
 
     const retrievedPromise1 = buffer.get(requestId1);
     const retrievedPromise2 = buffer.get(requestId2);
     assert(retrievedPromise1 !== undefined);
-    assert(retrievedPromise2 === undefined);
+    assert(retrievedPromise2 !== undefined);
```

Comment:
You should use the method, it gives more context in case the assertion fails.
```suggestion
      assert.notStrictEqual(retrievedPromise2, undefined);
```

Reply:
Changed here and in all other places, where I've used triple-equals - df824052. Thanks for pointing that out.